github/resource_github_repository_autolink_reference.go

```diff
@@ -20,12 +18,12 @@ func resourceGithubRepositoryAutolinkReference() *schema.Resource {
 
 		Importer: &schema.ResourceImporter{
 			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-				parts := strings.Split(d.Id(), "/")
-				if len(parts) != 2 {
-					return nil, fmt.Errorf("Invalid ID specified. Supplied ID must be written as <repository>/<autolink_reference_id>")
+				repoName, autolinkRefID, err := parseTwoPartID(d.Id(), "repository", "autolink_reference_id")
+				if err != nil {
+					return nil, err
 				}
-				d.Set("repository", parts[0])
-				d.SetId(parts[1])
+				d.Set("repository", repoName)
+				d.SetId(autolinkRefID)
```

Comment:
Use `parseTwoPartID` func from `util.go` for consistent import id structure parsing across the project.

Reply:
Reverted this change to avoid introducing a breaking change and requiring a major version bump. Will explore a separate PR that is focused only on standardizing the import syntax for all resources.